src/settings/settings_tab.ts

```diff
@@ -340,6 +340,19 @@ export class LatexSuiteSettingTab extends PluginSettingTab {
 				.setValue(this.plugin.settings.taboutEnabled)
 				.onChange(async (value) => {
 					this.plugin.settings.taboutEnabled = value;
+
+					await this.plugin.saveSettings();
+				}));
+
+		new Setting(containerEl)
+			.setName("Closing brackets")
+			.setDesc("A list of closing brackets for tabout, separated by commas.")
+			.addText(text => text
+				.setPlaceholder(DEFAULT_SETTINGS.taboutClosingSymbols)
+				.setValue(this.plugin.settings.taboutClosingSymbols)
+				.onChange(async (value) => {
+					this.plugin.settings.taboutClosingSymbols = value;
+
 					await this.plugin.saveSettings();
 				}));
```

Comment:
```suggestion
		const taboutClosingBracketsSetting = new Setting(containerEl)
			.setName("Closing brackets")
			.setDesc("A list of closing brackets for tabout, separated by commas.")
			.addText(text => text
				.setPlaceholder(DEFAULT_SETTINGS.taboutClosingSymbols)
				.setValue(this.plugin.settings.taboutClosingSymbols)
				.onChange(async (value) => {
					this.plugin.settings.taboutClosingSymbols = value;

					await this.plugin.saveSettings();
				}));
		taboutClosingBracketsSetting.settingEl.toggleClass("hidden", !this.plugin.settings.taboutEnabled);
		taboutEnabledSetting.addToggle((toggle) => toggle
			.setValue(this.plugin.settings.taboutEnabled)
			.onChange(async (value) => {
				this.plugin.settings.taboutEnabled = value;
				taboutClosingBracketsSetting.settingEl.toggleClass("hidden", !value);
				await this.plugin.saveSettings();
			})
		);
```
something like this (vscode doesn't let me select the whole section for some reason and don't want to figure out how for now)